app/controllers/hiring_staff/vacancies/application_controller.rb

```diff
@@ -139,4 +132,8 @@ def add_errors_to_form(errors, form_object)
       form_object.errors.add(field, error)
     end
   end
+
+  def redirect_unless_school_group_user_flag_on
+    redirect_to job_specification_organisation_job_path(request.parameters) unless SchoolGroupJobsFeature.enabled?
```

Comment:
Could we be explicit about what we're passing here (i.e `school_id: id` or `school_group_id: id`)? It feels wrong to blindly pass through all request parameters

Reply:
I'm not sure what parameters will be used. My intention was to redirect any requests to a path blocked by the flag, and redirect the parameters whatever they may be. Should I explicitly define what parameters are possible?

Reply:
I agree with Connor here

Reply:
Where should this redirect, if the flag is on? I don't know how the person would find themselves on this page if the flag is on. Maybe just redirect to homepage?

Reply:
We need to agree an approach, but the most sensible place to redirect - from my perspective - would be the dashboard